### PR TITLE
Create GET route/endpoint

### LINKED ISSUE #78: Create a route endpoints for retrieving all questions

### PR DESCRIPTION
**Pull request by abdulkadiret**

### CODE REVIEW

**Comment by bonboh** [comment]
I merged #72 to master which has resulted in conflicts here. Can you update with the latest master?

**Comment by abdulkadiret** [comment]
I fixed the merge conflict

File: routes/quizzes.js [hunk 1]
```diff
@@ -0,0 +1,16 @@
+const express = require("express");
```
Comment (bonboh) [inline routes/quizzes.js]: We're using Prettier which formats with single quotes — is your IDE working with Prettier correctly? The code should be auto-formatted on save

File: routes/quizzes.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+const router = express.Router();
+const Quiz = require("../models/quiz");
```
Comment (bonboh) [inline routes/quizzes.js]:
```suggestion
const quiz = require("../models/quiz");
```
The convention for JS variable names are that they are only capitalised if they are classes, are instantiable (i.e. you can do `new Quiz()`, or are components

**Review by bonboh — CHANGES_REQUESTED** [review]
Good work! Just a couple of small suggestions. Can you also update to the latest master to resolve the conflicts?

File: routes/quizzes.js (hunk 2 above)
Comment (bonboh) [reply routes/quizzes.js]: I just saw in #80 that we are doing `new Quiz` so we can leave it as `const Quiz`

**Comment by abdulkadiret** [comment]
Done! :)

File: routes/quizzes.js (hunk 1 above)
Comment (abdulkadiret) [reply routes/quizzes.js]: After I update with the latest master with Vicky's latest changes It's working fine.

**Review by bonboh — APPROVED** [review]
Looks fine to me 👍

**Pull request merged**